src/actions/importing/aioseo/abstract-aioseo-settings-importing-action.php

```diff
@@ -190,7 +201,7 @@ public function index() {
 	protected function query( $limit = null ) {
 		$aioseo_settings = \json_decode( \get_option( $this->get_source_option_name(), '' ), true );
 
-		if ( empty( $aioseo_settings ) || ! isset( $aioseo_settings['searchAppearance'][ $this->settings_tab ] ) ) {
```

Comment:
To double check, are all cases of removing this condition handled by the `isset_settings_tab` function being called beforehand or should that call be included here?

Reply:
The former :) 

Since we now do the isset() in a previous step/action, I found it would be redundant to check it again here